docs/pages/application-access/getting-started.mdx

```diff
@@ -4,11 +4,24 @@ description: Getting started with Teleport application access.
 videoBanner: 5Uwhp3IQMHY
 ---
 
-Let's connect to Grafana using Teleport in three steps:
+This tutorial demonstrates how to configure secure access to an application through Teleport. 
+The tutorial uses Grafana as a sample application because it's straightforward to install and run 
+in a Docker container or Kubernetes cluster with no additional configuration required.
+If you want to configure access for a different web application, you can use this tutorial as a 
+general guide for what to do. 
 
-- Launch Grafana in a Docker container or Kubernetes cluster.
-- Install the Teleport Application Service and configure it to proxy Grafana.
-- Access Grafana through Teleport.
+At a high level, configuring access for applications involves the following steps:
+
+* Verify your environment meets the prerequisites.
+* Verify you can launch the application in a Docker container, Kubernetes cluster, or using another method.
+* Generate a short-lived token for the application.
```

Comment:
The short-lived token is a join token for the Teleport Application Service to join the Teleport cluster.